pkgs/development/compilers/tinycc/default.nix

```diff
@@ -1,50 +1,31 @@
 { lib
 , stdenv
-, fetchFromRepoOrCz
-, copyPkgconfigItems
-, makePkgconfigItem
+, fetchFromGitHub
 , perl
 , texinfo
 , which
 }:
 
 stdenv.mkDerivation rec {
   pname = "tcc";
-  version = "unstable-2022-07-15";
+  version = "0.9.27+date=2022-05-26";
+  # clang ld requires a 32-bit x.y.z version number
+  cleanVersion = builtins.elemAt (lib.splitString "+" version) 0;
 
-  src = fetchFromRepoOrCz {
+  # move back to fetchFromRepoOrCz if they resolve their hosting/networking issues
+  src = fetchFromGitHub {
+    owner = "tinycc";
     repo = "tinycc";
-    rev = "af1abf1f45d45b34f0b02437f559f4dfdba7d23c";
-    hash = "sha256-jY0P2GErmo//YBaz6u4/jj/voOE3C2JaIDRmo0orXN8=";
+    rev = "afc136262e93ae85fb3643005b36dbfc30d99c42";
+    hash = "sha256-1XXVyvtBzSxlNn0wPYfKYAPkJqb1wx2IHIm2snnfUrM=";
   };
 
   nativeBuildInputs = [
-    copyPkgconfigItems
     perl
     texinfo
     which
   ];
 
-  pkgconfigItems = [
```

Comment:
Is this failing?

Reply:
I think this was dropped in a bad rebase since im not familiar with this chunk of code...
Ill need to add it back in and check